returnn/datasets/basic.py

```diff
@@ -245,6 +248,13 @@ def _get_default_random_seed_offset():
         config = get_global_config(raise_exception=False)
         if not config:
             return 0
+
+        env_val = os.environ.get(RANDOM_SEED_OFFSET_ENV_VAR)
+        if env_val is not None:
+            try:
+                return int(env_val)
+            except ValueError:
+                pass
```

Comment:
If it is set (non empty string), I think it should not silently ignore invalid values but raise an exception instead. I.e. simply like this:
```suggestion
        if env_val:
            return int(env_val)
```

Reply:
ah, fair point. will do